packages/legend-art/style/base/_panel.scss

```diff
@@ -203,6 +203,10 @@
   .input--small {
     max-width: 45rem;
   }
+
+  .input--small--set {
```

Comment:
I'm as hesitant to add this class as much as the struggle you have to name it, so maybe let's try something like setting the input field to `width: 100%` and used together with `input--small`

Reply:
Yes width:100% will still not change input and keep it at min length since its inside PanelListItem component so that both items in one row, but I added it as `&--full` hopefully that makes more sense?